docs/Installation/InstallationOnClusters.md

```diff
@@ -62,3 +62,11 @@ To run the tests:
 4. Run `cmake -D CMAKE_BUILD_TYPE=[Release|Debug]
    -D CMAKE_Fortran_COMPILER=gfortran $SPECTRE_HOME`
 5. Run `make -j4`
+
+## Minerva at AEI (WIP)
+
+1. Clone SpECTRE into `$SPECTRE_HOME`
+2. `source $SPECTRE_HOME/support/Environments/minerva.env` (WIP)
```

Comment:
no WIP, either it works or it doesn't. If it doesn't don't include it. You should also base it off the BlueWaters style with `specter_load_modules`, `spectre_run_cmake`, etc. functions. The SpEC-style env files are really annoying because there is no way to revert a load of it other than logging out. You probably looked at the Wheeler one, which I haven't updated yet, sorry.

Reply:
It works, just haven't pushed the `.env` yet. It only loads a bunch of Spack modules. Will model it after your BlueWaters setup then >)

Reply:
Okay, want to add the `env` (rename to `.sh`) in this PR? Then I'm happy for this to go in :)

Reply:
I'll open another PR with the Minerva setup ok? We should upgrade to `catch v2` first.

Reply:
No, just add it to this. We already require Catch v2

Reply:
Ah okay, so I'd like to see that upgrade happen and the env file be added here, since we only want systems we actually support listed here.

Reply:
Yep, removed the section for now and will open another PR when we did the Minerva upgrade.

Reply:
Okay, thank you :)

I'm at a workshop over the next few days, and then a conference so I may not be able to check PRs until the evening or mornings, just so you know I'm not ignoring you :)